src/screens/signup/signup3.tsx

```diff
@@ -24,11 +25,11 @@ interface SignupScreenThreeProp extends NavigationInterface {
 export default function SignUpScreenThree(props: SignupScreenThreeProp) {
   const { colors, fonts } = useThemeContext();
   const [name, setName] = useState('Claire Tamara');
-  const [password, setPassword] = useState('')
+  const [password, setPassword] = useState('');
   return (
-    <Container>
-      <SafeAreaView>
-      <NavigationView>
+    <SafeAreaView>
+      <Container>
+        <NavigationView>
           <TouchableOpacity onPress={() => props.navigation.goBack()}>
```

Comment:
For header navigation please use the custom app header that works for all screens as this code won't work efficiently for android notch screens

the custom header can be found inside the custom folder and its implementations can be found on regiment screen or settings screen even haircare screen

Reply:
Done, I have made all necessary changes